__tests__/handle-pull-request-change.js

```diff
@@ -32,6 +32,20 @@ describe('handlePullRequestChange', () => {
     expect(mock.isDone()).toBe(true)
   })
 
+  describe('when `enabled` is set to `false` in config', () => {
+    test('does not set a status check', async () => {
+      const context = buildContext()
+      context.payload.pull_request.title = 'do a thing'
+
+      const mock = nock('https://api.github.com')
+        .get('/repos/sally/project-x/contents/.github/semantic.yml')
+        .reply(200, getConfigResponse(`enabled: false`))
+
+      await handlePullRequestChange(context)
+      expect(mock.isDone()).toBe(true)
+    })
+  })
```

Comment:
I'm not familiar with this testing library... Is there a better way to test to ensure the status check API is _not_ called?

Reply:
I think you want to test here that it _does_ set a status check, that it passes, and that the message matches your expectations, e.g. `Check disabled in semantic.yml config`